docs/Lua API.rst

```diff
@@ -4079,10 +4079,22 @@ DFHack core invokes the scripts in the *core context* (see above);
 however it is possible to call them from any lua code (including
 from other scripts) in any context, via the same function the core uses:
 
-* ``dfhack.run_script(name[,args...])``
-
+* ``dfhack.run_script(command[, ...])``
+   
   Run a lua script in hack/scripts/, as if it was started from dfhack command-line.
-  The ``name`` argument should be the name stem, as would be used on the command line.
```

Comment:
I'd suggest keeping `name` (see other comment) because it is the name of a script specifically, not an arbitrary command. Perhaps "name stem" could be replaced with "name of the script without the `.lua` extension" for clarity, though.